### PR TITLE
Fixed old OVA url (1.0.4)

### PR DESCRIPTION
**Pull request by hseow-seagate**
Signed-off-by: Harrison Seow <harrison.seow@seagate.com>

### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [ ] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [ ] tested locally
 - [ ] added new dependencies
 - [x] updated the docs
 - [ ] added a test

-----
[View rendered CONTRIBUTING.md](https://github.com/hseow-seagate/cortx/blob/patch-12/CONTRIBUTING.md)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 CORTX is about building the world's best scalable mass-capacity object storage system. If you’re interested in what we’re building and intrigued by hard challenges, here's everything you need to know about contributing to the project and how to get started. 
 
-This guide is intended to provide quick start instructions for developers who want to build, test, and contribute to the CORTX software.  If you are merely looking to _test_ CORTX, please use [these instructions](doc/ova/1.0.4/CORTX_on_Open_Virtual_Appliance.rst).
+This guide is intended to provide quick start instructions for developers who want to build, test, and contribute to the CORTX software.  If you are merely looking to _test_ CORTX, please use [these instructions](doc/ova/2.0.0/PI-3/CORTX_on_Open_Virtual_Appliance_PI-3.rst).
```
Comment (mukul-seagate11) [inline CONTRIBUTING.md:5]: @hseow-seagate, you can update the latest OVA link as published in https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-4/CORTX_on_Open_Virtual_Appliance_PI-4.rst

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (hseow-seagate) [reply CONTRIBUTING.md]: Done

Comment (mukul-seagate11) [reply CONTRIBUTING.md]: updated link is broken

Comment (hseow-seagate) [reply CONTRIBUTING.md]: Hi @mukul-seagate11, the updated link is not broken. I did not update branch that's why its broken when trying to access the link from patch-12
I have now updated branch to patch-12. The link works fine now

**Comment by hseow-seagate** [comment]
@mukul-seagate11 So is this good to go or there's gonna be a new release again?

**Comment by mukul-seagate11** [comment]
> @mukul-seagate11 So is this good to go or there's gonna be a new release again?


> @mukul-seagate11 So is this good to go or there's gonna be a new release again?

yes, there will be a new release but now it will be based on K8s design

**Review by mukul-seagate11 — APPROVED** [review]
looks good

**Pull request merged**